images/linux/scripts/SoftwareReport/SoftwareReport.Helpers.psm1

```diff
@@ -0,0 +1,52 @@
+function Get-CommandResult {
+    param (
+        [Parameter(Mandatory=$true)]
+        [string] $Command,
+        [switch] $Multiline
+    )
+    # Bash trick to suppress and show error output because some commands write to stderr (for example, "python --version")
+    $stdout = & bash -c "$Command 2>&1"
+    $exitCode = $LASTEXITCODE
+    return @{
+        Output = If ($Multiline -eq $true) { $stdout } else { [string]$stdout }
+        ExitCode = $exitCode
+    }
+}
+
+function Take-Part {
```

Comment:
I don't like this function name since it doesn't look clear. But I can't suggest better name.
cc: @AlenaSviridenko do you have any ideas?

Reply:
@maxim-lobanov `Take-OutputPart` or `Take-SelectedPart` or `Split-ThenTakePart`?

Reply:
`Split-StringThenGetElementByIndex`

Reply:
`Parse-Element` or `Parse-SubString`

Reply:
I like `Take-OutputPart`. so let's leave as it is or use `Take-OutputPart`